lib/Checkout/Four/CheckoutApi.php

```diff
@@ -28,6 +32,12 @@ public function __construct(ApiClient $apiClient, CheckoutConfiguration $configu
         $this->instrumentsClient = new InstrumentsClient($apiClient, $configuration);
         $this->forexClient = new ForexClient($apiClient, $configuration);
         $this->disputesClient = new DisputesClient($apiClient, $configuration);
+        $this->sessionsClient = new SessionsClient($apiClient, $configuration);
+        $fileApiConfig = $configuration->getFilesConfiguration();
+        if ($fileApiConfig != null) {
```

Comment:
I'm not sure if this is the right sport for this validation. MarketplaceClient should be available either if we provide a `files` environment or not. Because only the uploadFile operation depends on fileApiConfig.

Said that, if a Customer wants to use MarketPlace will receive a `undefined`?

Reply:
You're right!